superwall/src/main/java/com/superwall/sdk/paywall/presentation/internal/operators/GetPresenter.kt

```diff
@@ -94,13 +95,34 @@ suspend fun Superwall.getPresenterIfNecessary(
     return currentActivity
 }
 
-private suspend fun Superwall.activateSession(
+suspend fun Superwall.attemptTriggerFire(
     request: PresentationRequest,
     triggerResult: InternalTriggerResult,
-): String? {
+) {
+    if (request.flags.type == PresentationRequestType.GetImplicitPresentationResult ||
+        request.flags.type == PresentationRequestType.GetPresentationResult
+    ) {
+        return
+    }
```

Comment:
I think this is different than the iOS implementation

Reply:
Yes, picked it up from the `GetExperiment` by mistake, removed!